README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget App
+> Budget App: Show spending and transactions history. Users can create a category, an add transactions to it.
+
+## Built With
+- Rails
+- CSS
+- PostgreSQL
+
+## Online Version Link
+This project is available online through the following link.
+https://sheltered-cove-77932.herokuapp.com/
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A text editor eg Atom
+- Ruby
+- Rails
+
+### Setup
+
+ **Step 1: Clone this repo locally using git**
+- Create a folder in your local machine
+- Open your console in your folder and run this command: `git clone https://github.com/NduatiKagiri/budget-app.git`
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+ **Step 2: Run in your project directory**
+- Run `bundle install` to install all the required gems
+- Run `rails server` to create a server for the project
+
+## Authors
+:bust_in_silhouette: **NduatiKagiri**
+
+GitHub: https://github.com/NduatiKagiri
+Twitter: https://twitter.com/NduatiKagiri
+LinkedIn: https://linkedin.com/in/nduati-kagiri/
+
+## :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
```

Comment:
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```